src/components/pastAnalysis/PastAnalysisTable.tsx

```diff
@@ -172,23 +173,41 @@ const PastAnalysisTable = () => {
   }
 
   const updateRows = (listOfAnalysis: StudyInstanceWithSeries[]) => {
-    const rows: (tableRowsChild | tableRowsParent)[] = []
+    const rows: (tableRowsChild | tableRowsParent)[] = [];
     for (const analysis of listOfAnalysis) {
+      const validAnalyses = analysis.series.filter((series: ISeries) => series.classifications.size > 0);
+      const classifications = validAnalyses[0]?.classifications ? Array.from(validAnalyses[0]?.classifications?.keys()) : [];
+      const numInvalidAnalyses = analysis.series.length - validAnalyses.length;
+
       const indexInRows: number = rows.length;
+
+      const isProcessing = analysis.analysisCreated === Processing.analysisAreProcessing;
+      let analysisCreated;
+      let badges;
+      if (isProcessing) {
+        analysisCreated = {
+          title: (<div><Spinner size="md" /> Processing</div>)
+        };
+        badges = '';
+      } else {
+        analysisCreated = analysis.analysisCreated;
+        badges = {
+          title: (<>
+            {<Badge className="badge-margin" isRead={!validAnalyses.length}>{validAnalyses.length}</Badge>}
+            {<Badge style={numInvalidAnalyses ? { backgroundColor: "var(--pf-global--danger-color--100)" }: {}} isRead={!numInvalidAnalyses}>{numInvalidAnalyses}</Badge>}
```

Comment:
I highly suggest using conditional classes instead of conditional styling. It's better to not use any inline styling and keep all the styling in CSS/SCSS (Separation of Concern Rule)

Reply:
Unfortunately, we aren't able to override the PatternFly default color easily by using a class. The only options we are to use inline styles or the `!important` flag. Which do you think is better?

Reply:
If you apply the styling using a more specific CSS selector, you will not need to use `!important`.

As an example, you could use `.pf-c-badge.pf-m-unread.badge-warn`.